### DISCUSSION TITLE
Sprite numbers overlap

### DISCUSSION THREAD

**Discussion by mike-rankin** [Bodmer/TFT_eSPI#2977] [General]
Is there a recommended way to prevent sprint numbers from over writing on top of the previous? Right now I can refresh the entire screen or draw a black box over just the numbers but was wondering if there is a better way?

Here is par tof my code:

```
void setup(void)
 {
  pinMode(TFT_CS, OUTPUT);
  digitalWrite(TFT_CS, LOW); 	 
  tft.init();
  tft.setRotation(1);
  sprite.createSprite(280,240);
  tft.fillScreen(TFT_BLACK);
}

void loop() 
{
  String text;
  sprite.setTextColor(White, Black); 
  sprite.loadFont(bigFont);

  for(int i=0;i<10;i++)
  {
   text=i;
   sprite.setTextColor(White, Black);   
   sprite.drawString(text,50,100);
   delay(1000);
   sprite.pushSprite(0, 0);
  }
}
```
![text](https://github.com/Bodmer/TFT_eSPI/assets/4991664/7822f753-acda-474b-9dad-d550f1adc57f)

**Comment by Bodmer** [comment]
See [News in ReadMe](https://github.com/Bodmer/TFT_eSPI#news), item 8.
